mmf/datasets/builders/vqacp_v2/database.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+import copy
+import json
+import os
+
+from mmf.datasets.databases.annotation_database import AnnotationDatabase
+from mmf.datasets.databases.image_database import (
+    ImageDatabase,
+    get_possible_image_paths,
+)
+from mmf.utils.file_io import PathManager
+
+
+class VQACPv2AnnotationDatabase(AnnotationDatabase):
```

Comment:
Inherit the OKVQAAnnotationDatabase to reduce duplication.

Reply:
Done, but I still have to override the ` load_annotation_db` function since the format of the annotations is slightly different. So the code size hasn't reduced by much.

> The format of the questions is same as that of the VQA v1 dataset except that this file only contains the "questions" field